src/YQWizard.cc

```diff
@@ -427,7 +427,7 @@ void YQWizard::updateSteps()
     rbl->addWidget( (QWidget *) _releaseNotesButton->widgetRep(), 0, Qt::AlignCenter );
 
     _stepsVBox->addLayout( rbl );
-    _stepsVBox->addStretch( 29 );
+    _stepsVBox->addStretch( 1 );
```

Comment:
BTW what was the problem with this stretch value?

Reply:
Button was higher than the bottom edge, simply a stylistic choice so it looked more in-line with other buttons (still will have to be adjusted in style though, because seems that there is a frame around other widgets which makes it go out of alignment). With lower value, I can fix the rest in the stylesheet.
![screenshot from 2018-10-16 07-41-52](https://user-images.githubusercontent.com/30577011/47014051-60b39600-d149-11e8-87b6-c4d6c336b203.png)

Reply:
I see, thanks for the screenshot!